ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -82,21 +89,24 @@ S3CustomizeFileSystem(WT_STORAGE_SOURCE *storageSource, WT_SESSION *session, con
     aws_config.throughputTargetGbps = throughputTargetGbps;
     aws_config.partSize = partSize;
 
+    Aws::Utils::Logging::InitializeAWSLogging(
+      Aws::MakeShared<S3LogSystem>("storage", s3->wtApi, s3->awsVerboseLevel));
```

Comment:
Not sure here what happens if make_shared were to fail. There doesn't appear to be any error handling.